architecture/service_catalog/service_catalog_cli.adoc

```diff
@@ -0,0 +1,434 @@
+----
+
+[[service-catalog-cli-provision-services]]
+=== Provision services
+Provisioning means to make the service available for consumption. To provision a
+service, you need to create a service instance and then bind to it.
+
+[[service-catalog-cli-create-serviceinstance]]
+==== Create ServiceInstance
+
+[NOTE]
+====
+Service instances must be created inside an OpenShift namespace.
+====
+
+. Create a new project.
++
+[source, bash]
+----
+$ oc new-project <project-name> <1>
+----
+<1> Replace `<project-name>` with name of your project.
+
+. Create a service instance definition file *_postgresql-apb.yaml_*.
++
+[source, bash]
+----
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: postgresql
+spec:
+  clusterServiceClassExternalName: dh-postgresql-apb
+  clusterServicePlanExternalName: dev
+  parameters:
+    postgresql_database: "admin"
+    postgresql_password: "admin"
+    postgresql_user: "admin"
+postgresql_version: "9.6"
+----
+
+. Create service instance using the command:
++
+[source, bash]
+----
+$ oc create -f postgresql-apb.yaml
+----
+
+[[service-catalog-cli-view-serviceinstance]]
+===== View service instance details
+To view service instance details, run the command:
+[source, bash]
+----
+$ svcat get instances
+  NAME   NAMESPACE   CLASS   PLAN   STATUS
++------+-----------+-------+------+--------+
+
+----
+
+[[service-catalog-cli-create-servicebinding]]
+==== Create ServiceBinding
+[NOTE]
+====
+When you create a `ServiceBinding` resource:
+
+. The service catalog controller communicates with the broker server to initiate the binding.
+. The broker server create credentials and issue them to the service catalog controller.
+. The service catalog controller adds those credentials as secrets to the project.
+====
+
+. Create a service binding definition file *_postgresql-apb-mediawiki-apb-bind.yaml_*
++
+[source, bash]
+----
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  name: mediawiki-postgresql-binding
+spec:
+  instanceRef:
+    name: postgresql
+secretName: mediawiki-postgresql-binding
+----
+
+. Create the service binding using the command
++
+[source, bash]
+----
+$ oc create -f postgresql-apb-mediawiki-apb-bind.yaml
+----
+
+[[service-catalog-cli-view-servicebinding]]
+===== View service binding details
+To view service binding details, run the command:
+[source, bash]
+----
+$  svcat get bindings
+  NAME   NAMESPACE   INSTANCE   STATUS
++------+-----------+----------+--------+
+
+----
+
+[[service-catalog-cli-deleting-resources]]
+== Deleting resources
+Before you can delete resources, you need to unbind and deprovision the service
+instances.
+
+[[service-catalog-cli-delete-servicebindings]]
+=== Deleting service bindings
+
+. Unbind the service instance:
++
+[source, bash]
+----
+$ svcat unbind -n <project-name> <instance-name>
+----
+
+. Delete the service bindings:
++
+[source, bash]
+----
+$ oc delete -n <project-name> servicebindings <binding-name>
```

Comment:
Unbind the service instance is ok, we don't need to delete the service binding in step 2.  
```
$ svcat unbind --name mediawiki-postgresql-binding
deleted mediawiki-postgresql-binding
```